airbyte-integrations/connectors/source-okta/integration_tests/configured_catalog.json

```diff
@@ -4,7 +4,8 @@
       "stream": {
         "name": "users",
         "json_schema": {},
-        "supported_sync_modes": ["full_refresh"]
+        "supported_sync_modes": ["full_refresh"],
+        "supported_sync_modes": ["full_refresh", "incremental"]
```

Comment:
Now you have 2 `supported_sync_modes` :)

Reply:
updated